app/models/physical_server_profile_template.rb

```diff
@@ -0,0 +1,63 @@
+class PhysicalServerProfileTemplate < ApplicationRecord
+  acts_as_miq_taggable
+
+  include NewWithTypeStiMixin
+  include TenantIdentityMixin
+  include SupportsFeatureMixin
+  include EventMixin
```

Comment:
Do we have relationships from `EventStream` to `PhysicalServerProfileTemplate`?  I don't think so we should probably leave this mixin out

Reply:
I deleted this mixin.